publication/src/main/scala/hmda/publication/regulator/ts/RegulatorTsPublisher.scala

```diff
@@ -0,0 +1,100 @@
+package hmda.publication.regulator.ts
+
+import java.time.LocalDateTime
+
+import akka.actor.{ ActorSystem, Props }
+import akka.http.scaladsl.model.{ ContentType, HttpCharsets, MediaTypes }
+import akka.stream.{ ActorMaterializer, ActorMaterializerSettings, Supervision }
+import akka.stream.Supervision.Decider
+import akka.stream.alpakka.s3.impl.{ S3Headers, ServerSideEncryption }
+import akka.stream.alpakka.s3.javadsl.S3Client
+import akka.stream.alpakka.s3.{ MemoryBufferType, S3Settings }
+import akka.stream.scaladsl.Source
+import akka.util.ByteString
+import com.amazonaws.auth.{ AWSStaticCredentialsProvider, BasicAWSCredentials }
+import com.typesafe.akka.extension.quartz.QuartzSchedulerExtension
+import hmda.persistence.model.HmdaActor
+import hmda.publication.regulator.messages.PublishRegulatorData
+import hmda.query.repository.filing.TransmittalSheetCassandraRepository
+
+object RegulatorTsPublisher {
+  def props(): Props = Props(new RegulatorTsPublisher)
+}
+
+class RegulatorTsPublisher extends HmdaActor with TransmittalSheetCassandraRepository {
```

Comment:
I made this comment previously, but it looks like it got eaten: It looks like `ModifiedLarPublisher`, `RegulatorLarPublisher`, `RegulatorPanelPublisher`, and `RegulatorTsPublisher` duplicate a bunch of configuration code. Could they share that code in a common module instead?

Reply:
Yeah, forgot to reply to that comment. They could, but I encountered some issues while doing that refactor and decided against it at this time